sententia/src/main/java/nl/nfi/sententia/SententiaPlugin.java

```diff
@@ -170,10 +171,13 @@ public void domainObjectChanged(DomainObjectChangedEvent event) {
 		Iterator<DomainObjectChangeRecord> it = event.iterator();
 		while (it.hasNext()) {
 			DomainObjectChangeRecord evt = it.next();
-			if (evt.getEventType().getId() == ProgramEvent.SYMBOL_RENAMED.getId()) {				
+			// Check if we're dealing with a rename and the is applied to a function
+			if (evt.getEventType().getId() == ProgramEvent.SYMBOL_RENAMED.getId() && ((ProgramChangeRecord)evt).getObject() instanceof FunctionSymbol) {				
```

Comment:
Not familiar with Ghidra's API, I'm assuming this is the recommended way of checking / filtering this? :)

Reply:
I think so, but the API docs are pretty meager. Seems to be the most specific solution to this problem though.